config/config_test.go

```diff
@@ -1500,6 +1500,11 @@ var expectedConf = &Config{
 	},
 }
 
+func TestYAMLNotLongerSupportedAMApi(t *testing.T) {
+	_, err := LoadFile("testdata/config_with_no_longer_supported_am_api_config.yml", false, false, promslog.NewNopLogger())
```

Comment:
@alanprot  This interface changed in the meantime. Almost good to go :sweat_smile: 
```suggestion
	_, err := LoadFile("testdata/config_with_no_longer_supported_am_api_config.yml", false, promslog.NewNopLogger())
```